src/components/LeafletMap.js

```diff
@@ -247,4 +249,20 @@ export class LeafletMap {
       dispatch.call("hide-loading-indicator");
     }
   };
+  findAutocompletLocation(resource) {
+    let center = resource.point.coordinates;
+    const markerIcon = L.icon({ iconUrl: "assets/empty-icon.svg" });
```

Comment:
Reiterating the comment left on the `empty-icon.svg` file, would it be better to use Leaflet's default marker icon?

Reply:
changed to default marker